### PR TITLE
Mnd 13605/webauthn QR fix

### PR DESCRIPTION
**Pull request by Idan-Berman**

### CODE REVIEW

File: passkey-authentication/pages/home.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <title>ACME</title>
     <script
       type="text/javascript"
-      src="https://cdn.identity.security/authentication-sdk/2.10.0/TsAuthSdk.min.js"
+      src="https://platform-websdk.idsec-dev.com/platform-websdk/1.3.3-bitbank/ts-platform-websdk.js"
```
Comment (dubek) [inline passkey-authentication/pages/home.html:14]:
```suggestion
      src="https://platform-websdk.transmitsecurity.io/platform-websdk/1.3.3-bitbank/ts-platform-websdk.js"
```

Comment (Idan-Berman) [reply passkey-authentication/pages/home.html:14]: fixed. thanks

File: webauthn-cross-device/pages/desktop.html [hunk 2]
```diff
@@ -91,6 +91,7 @@
           );
           window.pageUtils.hide('spinner');
         }
+        console.log(controller);
```
Comment (buenonadav) [inline webauthn-cross-device/pages/desktop.html:94]: Leftover?

**Review by buenonadav — APPROVED** [review]
LGTM

File: webauthn-cross-device/pages/desktop.html (hunk 2 above)
Comment (Idan-Berman) [reply webauthn-cross-device/pages/desktop.html]: removed

**Pull request merged**